imageio/imageio-tiff/src/main/java/com/twelvemonkeys/imageio/plugins/tiff/TIFFImageReader.java

```diff
@@ -821,6 +822,7 @@ private int getDataType(int sampleFormat, int bitsPerSample) throws IIOException
                 throw new IIOException("Unsupported BitsPerSample for SampleFormat 2/Signed Integer (expected 8/16/32): " + bitsPerSample);
```

Comment:
Should probably change this message now that we support more sample formats than INT/2: 

```java
throw new IIOException(String.format("Unsupported BitsPerSample for SampleFormat %d/Signed Integer (expected 8/16/32): %d", sampleFormat, bitsPerSample));
```